sql-plugin/src/main/spark311/scala/org/apache/spark/sql/rapids/ShimmedExecutionPlanCaptureCallbackImpl.scala

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*** spark-rapids-shim-json-lines
+{"spark": "311"}
+{"spark": "312"}
+{"spark": "313"}
+{"spark": "320"}
+{"spark": "321"}
+{"spark": "321cdh"}
+{"spark": "321db"}
+{"spark": "322"}
+{"spark": "323"}
+{"spark": "324"}
+{"spark": "330"}
+{"spark": "330cdh"}
+{"spark": "330db"}
+{"spark": "331"}
+{"spark": "332"}
+{"spark": "333"}
+spark-rapids-shim-json-lines ***/
```

Comment:
This should not be in the shim directories.  There's no source code in this class that is specific to a version of Spark.  The only things that should be in the shim directories and using the shim-json-lines are files that have at least two different source code versions.  If the source code is identical across all Spark versions but compiles to different bytecodes, the build system will automatically handle that without needing to tag it for shimplify.

Reply:
Ok will move that makes sense.

Reply:
This worked locally: https://github.com/NVIDIA/spark-rapids/pull/9113/commits/e17de2b43e019aee4fd2c027e51ddda3877fb409